packages/bootleg-app/src/drizzleOptions.js

```diff
@@ -1,13 +1,13 @@
-import BootlegTraderApp from 'bootleg-app-contracts/build/contracts/BootlegTraderApp.json';
-import BootlegToken from 'bootleg-app-contracts/build/contracts/BootlegToken.json';
+import BootlegTraderApp from './contracts/BootlegTraderApp';
```

Comment:
huh wha?! 😮

Reply:
Should we copy/paste when these change in the source project?

Reply:
I'm honestly not sure. This feels right to me because those build artifacts are giant and take a lot of bits. Only using the abi and hard coding connonical addresses seems good? Also, the abi rarely changes and unless we always deploy, the production abi needs to match the app. 

We have to check either the build assets or the abi in. This seemed more stable. CA:1 very open to ideas as all the ones I have I don't like here